dm/tests/dmctl_command/run.sh

```diff
@@ -196,20 +199,79 @@ function run_validator_cmd {
 		"\"stage\": \"Running\"" 2 \
 		"\"stage\": \"Stopped\"" 1 \
 		"no primary key" 1
-
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation show-errors --error all test" \
+		"\"id\": \"1\"" 1 \
+		"\"id\": \"2\"" 1 \
+		"\"id\": \"3\"" 1 \
+		"\"id\": \"4\"" 1
+	# resolve error 1
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation make-resolve test 1"
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"validation status test" \
+		"new\/ignored\/resolved: 3\/0\/1" 1
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
```

Comment:
maybe add a check for `--error resolved`

Reply:
`show-errors` can only query `unprocessed`(i.e. new), `all`, and `ignored` errors.